apps/src/studio/studio.js

```diff
@@ -2176,10 +2188,8 @@ var displayFeedback = function() {
       // allow users to save freeplay levels to their gallery (impressive non-freeplay levels are autosaved)
       saveToGalleryUrl: level.freePlay && Studio.response && Studio.response.save_to_gallery_url,
       message: Studio.message,
-      appStrings: {
-        reinfFeedbackMsg: studioMsg.reinfFeedbackMsg({backButton: tryAgainText}),
```

Comment:
I'm not even sure what it means, to expand it.  My best guess is "reinforcement" but I don't know what a reinforcement feedback message is in this context.